app/views/users/show.html.slim

```diff
@@ -0,0 +1,31 @@
+h2 =@user.full_name
+.user-profile.row 
+  .user-photo.col-md-3.col-xs-12
+    =image_tag(@user.image_url)
+  .user-info.col-md-9.col-xs-12
+    p Електронна пошта: #{@user.email}
+    p Запис зареєстровано: #{l @user.created_at, format: :long}
+- if @issues.any?
+  .box.user-issues.col-xs-12
+    h4 Мої звернення:
+    .table-responsive
+      table.table.table-hover
+        thead
+          tr
+            th Звернення
+            th Назва
+            th Дата створення
+            th Статус
+            th Посилання
+
+        tbody
+          - @issues.each do |issue|
+            tr
+              td № #{issue.id}
+              td =issue.title
+              td =l issue.created_at, format: :long
+              td 
+                span[class="label label-#{show_status(issue.status)[:label]}"] 
+                  =show_status(issue.status)[:text]
```

Comment:
* ```show_status``` helper should produce link and text:
  ```
  td
    = show_status issue
  ```

* If I'm not mistake, @igorkasyanchuk means that it would be good idea to add method like ```Issue#status_name``` that will return 'Запит прийнято', 'Очікує на модерацію' etc.